docs/how-to-deploy-bold-bi-and-configure-startup-manually.md

```diff
@@ -8,19 +8,7 @@ In the following section, we will deploy a single container Bold BI application
    
    ```sh
    curl -o docker-compose.yml "https://raw.githubusercontent.com/boldbi/boldbi-docker/main/deploy/single-container/docker-compose.yml"
-   ```
-3. Open the docker compose file and fill the mandatory field - <b>APP_URL</b>
-
-    ![docker-compose-database-detail](/docs/images/single-container-app-url.png)
-
-      <b>APP_URL Guidance:</b>
-      * Provide the HTTP scheme for APP_BASE_URL value.
-      For example, <br/>
-          `http://example.com` <br/>
-          `http://<public_ip_address>` <br/>
-      * For `Windows` and `MacOS` use either http://host.docker.internal or http://localhost. Docker Desktop provides `host.docker.internal` and `gateway.docker.internal` DNS for communication between docker applications and host machine. Please make sure that the host.docker.internal DNS has your IPv4 address mapped in your hosts file on Windows(C:\Windows\System32\drivers\etc\hosts).
-      * For `Linux` use the Machine Public IP address as the value for APP_URL with the HTTP scheme.
- 
+   ``` 
```

Comment:
You should use different compose file for each use case? Do we have a separate compose file with above mentioned screenshot?

Reply:
Yes, we have separate for this deployment.
<img width="705" alt="image" src="https://github.com/boldbi/boldbi-docker/assets/92422799/cf45d8e8-a201-4e15-94d9-36a5eda9b8a3">

Docker-compose yaml file=>https://raw.githubusercontent.com/SivanesanSyncfusion/boldbi-docker-old/main/deploy/single-container/docker-compose.yml

Reply:
Please let me know if you have any concern